opencast-backend/annotation-impl/src/test/java/org/opencast/annotation/endpoint/TestRestService.java

```diff
@@ -90,17 +87,12 @@ private static AuthorizationService getAuthorizationService() {
   private static SearchService getSearchService() {
     MediaPackage mediaPackage = EasyMock.createNiceMock(MediaPackage.class);
 
-    SearchResultItem searchResultItem = EasyMock.createNiceMock(SearchResultItem.class);
-    EasyMock.expect(searchResultItem.getMediaPackage()).andReturn(mediaPackage).anyTimes();
-    EasyMock.replay(searchResultItem);
-
-    SearchResult searchResult = EasyMock.createNiceMock(SearchResult.class);
-    EasyMock.expect(searchResult.getItems()).andReturn(new SearchResultItem[]{searchResultItem}).anyTimes();
-    EasyMock.replay(searchResult);
-
     SearchService searchService = EasyMock.createNiceMock(SearchService.class);
-    EasyMock.expect(searchService.getByQuery(EasyMock.anyObject(SearchQuery.class)))
-            .andReturn(searchResult).anyTimes();
+    try {
+      EasyMock.expect(searchService.get(EasyMock.anyObject(String.class))).andReturn(mediaPackage).anyTimes();
+    } catch (Exception e) {
```

Comment:
I would sleep ever so slightly better if we just caught whatever we have to so we don't hide any real exceptions.